performance/shopify/liquid.rb

```diff
@@ -1,4 +1,4 @@
-$:.unshift File.dirname(__FILE__) + '/../../lib'
+$LOAD_PATH.unshift File.dirname(__FILE__) + '/../../lib'
```

Comment:
If we're 2+ we could also relplace all these `File.dirname(__FILE__)` by `__dir__`. I doubt rubocop have something for that.